vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -2,570 +2,237 @@
+|`buildFolder`|String|Optional||Corresponds to the MFE build folder |
+
+#### Micro Frontends Sample Code
+```json
+ "microfrontends": [
+    {
+      "name": "mfe-name",
+      "stack": "react",
+      "titles": { "en": "en_mfe_title", "it": "it_mfe_title" }
+      "type": "app-builder",
+      "slot": "content",
+      "paths": ["/path1"],
+      "group" : "free",
+      "apiClaims": [
+          {
+              "name": "int-api-claim",
+              "type": "internal",
+              "serviceId": "ms-name"
+          },
+          {
+              "name": "ext-api-claim",
+              "type": "external",
+              "bundleId": "ext-bundle",
+              "serviceId": "ext-bundle-ms"
+          }
+      ],
+    },
+    {
+      "name": "mfe2-name",
+      "stack": "react",
+      "type": "widget",
+      "publicFolder": "public",
+      "titles": { "en": "en_mfe2_title", "it": "it_mfe2_title" },
+      "group": "free",
+      "commands": { "build": "custom-command" }
+    }
+```
+#### API Claim Specification
+|Name|Type|Validation|Possible Value|Description|
+|:-|:-|:-|:-|:------------------------|
+|`name`|String|Mandatory||Name|
+|`type`|Enum|Mandatory|*internal  *external|API claim specifying internal or external to bundle|
+|`serviceName`|String|Mandatory||Microservice name into the bundle|
+|`bundle`|String|Mandatory only for  `type=external`||Bundle Docker URL|
+* For an example, see the [Micro Frontends sample code](#micro-frontends-sample-code) above.
+
+#### Permission Specification
+|Name|Type|Validation|Description|
+|:-|:-|:-|:------------------------|
+|`clientId`|String|Mandatory|Client ID to assign|
+|`permission`|String|Mandatory|Roles to assign to the client ID|
+
+#### Command Specification
+|Name|Type|Validation|Description|
+|:-|:-|:-|:------------------------|
+|`build`|String|Optional|Custom build command|
+|`run`|String|Optional|Custom run command|
+
+#### Command Spec Sample Code
+```json
+"name": "another-ms",
+            "stack": "spring-boot",
+            "healthCheckPath": "/health",
+            "commands": {
+                "run": "mvn -Dspring-boot.run.arguments=\"--server.port=8082\" spring-boot:run"
+            },
+```
+
+#### MenuEntry Specification
+|Name|Type|Validation|Possible Values|Description|
+|:-|:-|:-|:-|:------------------------|
+|`label`||Mandatory||Localized entry on the PBC menu|
+|`target`||Mandatory|*internal  *external|Where to open the menu link|
+|`url`|String|||Address of the page to open when the menu is clicked|
+
+#### Environment Variables Specification
+|Name|Type|Validation|Description|
+|:-|:-|:-|:------------------------|
+|`name`|String|Mandatory|Name of the environment variable to inject|
+|`value`|String|Optional|Value to give to the environment variable|
+|`valueFrom`|SecretKeyRef|Optional|Reference to the Secret from which to fetch the value|
+
+### Troubleshooting
+1. **Support for Media/Images in Micro Frontends that are not in the Bundle**
```

Comment:
This isn't exactly a troubleshooting topic. It's more of a new feature/suggestion on how to build MFEs and I'm not quite sure where to place it. Let's discuss.